tests/tests_twobody/test_orbit.py

```diff
@@ -197,17 +197,6 @@ def test_parabolic_has_zero_energy():
     assert_allclose(ss.energy.value, 0.0, atol=1e-16)
 
 
-def test_pqw_for_circular_equatorial_orbit():
```

Comment:
We can't remove `Orbit.pqw` yet, because we need it for the plotting:

https://github.com/poliastro/poliastro/blob/49ae31844b2d59a6b052a42bc64f34efd0f870fb/src/poliastro/plotting/_base.py#L346-L357

We are tracking that in https://github.com/poliastro/poliastro/issues/557.

In this case, let's silence the deprecation warning inside the test.